rebar.config

```diff
@@ -5,9 +5,9 @@
 {ct_verbose, true}.
 
 {deps, [
-    {cowboy, {git, "https://github.com/K2InformaticsGmbH/cowboy", {tag, "1.1.2"}}},
-    {cowlib, {git, "https://github.com/K2InformaticsGmbH/cowlib", {tag, "1.0.2"}}},
-    {ranch, {git, "https://github.com/K2InformaticsGmbH/ranch", {tag, "1.3.2"}}}
+    {cowboy, {git, "https://github.com/K2InformaticsGmbH/cowboy", {tag, "2.0.0-rc.3"}}},
+    {cowlib, {git, "https://github.com/K2InformaticsGmbH/cowlib", {tag, "2.0.0-rc.1"}}},
+    {ranch, {git, "https://github.com/K2InformaticsGmbH/ranch", {tag, "1.4.0"}}}
```

Comment:
We are at a better version of cowboy already. Please upgrade:
![image](https://user-images.githubusercontent.com/913008/42128790-dba2391a-7cb3-11e8-9ba4-7550ff5cadc9.png)